pxr/imaging/hgiVulkan/device.cpp

```diff
@@ -69,7 +69,8 @@ _SupportsPresentation(
                 vkGetInstanceProcAddr(
                     vkInstance,
                     "vkGetPhysicalDeviceXlibPresentationSupportKHR");
-        return vkGetPhysicalDeviceXlibPresentationSupportKHR(
+        return vkGetPhysicalDeviceXlibPresentationSupportKHR &&
```

Comment:
Just FYI, this gives me a compilation error: `the address of 'VkBool32 vkGetPhysicalDeviceXlibPresentationSupportKHR(VkPhysicalDevice, uint32_t, Display*, VisualID)' will never be NULL`

Reply:
That's interesting, I actually observed this being null, when I used Lavapipe headless (didn't build it with X support). Is it a warning as an error or an actual error?

Reply:
It's a warning as an error. Do you know if there's some define we can use to branch this?

Reply:
Maybe the WSI macros? https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/WSIheaders.html

Reply:
`vkGetInstanceProcAddr` can return null, so I don't understand why the compiler would warn here.